docs/Developers/Backend/HostedSDK.md

```diff
@@ -11,3 +11,33 @@ The SDK is hosted at [https://api-v2.pendle.finance/sdk/](https://api-v2.pendle.
 Pendle accommodates a vast array of assets, each characterized by its unique nuances and complexities. While the Pendle protocol remains immutable, the underlying assets don't share this feature, requiring our app and SDK to be updated frequently to align with changes in these assets.
 
 To address this, Pendle has introduced a hosted version of our SDK. It ensures the output remains consistent with Pendle's UI and keeps up-to-date with the latest protocol changes. The API design prioritizes simplicity and stability, with a high rate limit to meet the needs of most users.
+
+# Hosted SDK (external)
+
+This SDK is hosted at [https://api-v2.pendle.finance/external/docs#/SDK](https://api-v2.pendle.finance/external/docs#/SDK).
+
+## Supported functions
+- Swap
+- Add liquidity
+- Remove liquidity
+- Mint PT & YT
+- Redeem PT & YT
+- Transfer liquidity
+- Roll over PT
+- Add liquidity dual
+- Remove liquidity dual
+
+**Example**
+
+To swap **1 USDC** to **PT eETH** in **eETH (0x7d372819240d14fb477f17b964f95f33beb4c704)** with 1% slippage:
+
+`GET https://api-v2.pendle.finance/external/v1/sdk/1/markets/0x7d372819240d14fb477f17b964f95f33beb4c704/swap?receiver=<RECEIVER_ADDRESS>&slippage=0.01&enableAggregator=true&tokenIn=0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48&tokenOut=0x6ee2b5e19ecba773a352e5b21415dc419a700d1d&amountIn=1000000`
```

Comment:
Also the URL is wrong. You missed `api` before the `external`.

Another reason to add the link :rofl: